subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/configuration/InstrumentationConfiguration.kt

```diff
@@ -104,6 +105,14 @@ open class InstrumentationConfiguration(val name: String) {
         val performanceType: PerformanceType?
     ) : Serializable {
 
+        init {
+            val hasLocal = targets.any { it.reservation.device is Device.LocalEmulator }
```

Comment:
Are you sure we had all the targets registered at that moment?

Reply:
Yes, we build this DTO will all data. In any case it must have a local state.